### ISSUE TITLE
Dev UX: How to preview `docs/site` in incremental mode

### ISSUE THREAD

**Issue by bajtos** [strongloop/loopback-next#1310]
## Description / Steps to reproduce / Feature proposal

When making changes to documentation files in `docs/site` folder, I'd like to preview the changes I am making.

## Current Behavior

I can run `npm run build:site` command to copy all sources and start Jekyll server.

However, this process does not support incremental updates. Whenever I make a change, all files have to be re-downloaded from `docs/site` and GitHub repositories.

Alternatively, it may be possible to combine `npm run copydocs`  and loopback.io's `npm start` to speed the updates a bit, but it's still not ideal.

## Expected Behavior

There should be a single monorepo-level npm script that will:

 1. Perform the initial copy of `docs/site`, download any assets from other GitHub repos.
 2. Either use symlinks or setup a filesystem watcher to automatically mirror changes from `docs/site` to local `loopback.io` clone
 3. Start jekyll in a watching mode, preferably with `--incremental` flag, so that any updates in `docs/site` are quickly applied to the local website.

Alternatively, we may want to look for a different solution that will allow us to render the content in `docs/site` without the rest of loopback.io content. What's important: we need to configure Jekyll the same way as it's configured on loopback.io, including plugins, CSS stylesheets, etc.

_See [Reporting Issues](http://loopback.io/doc/en/contrib/Reporting-issues.html) for more tips on writing good issues_

**Comment by bajtos** [comment]
While working on https://github.com/strongloop/loopback-next/issues/3718, I found this issue is much worse than I thought. We are already running Jekyll in an incremental mode, but it does not seem to pick changes :(

I ended up doing a full restart whenever I want to preview my updates. Here is the command that works for me, execute it in `sandbox/loopback.io` directory:

```sh
$ cp -r ../../docs/site/* pages/en/lb4/ && cp -r ../../docs/site/sidebars/* _data/sidebars/ && npm start
```

**Comment by emonddr** [comment]
I encountered problems with this before. I enlisted the help of Biniam and Raymond. Raymond made changes somewhere (can't remember what PR it was). It seemed to work for him, but never for me.
Cannot remember the actual file that @raymondfeng  updated in an attempt to fix the problem for me. It was in a program that copies documentation from one directory to another directory (I am assuming the sandbox directory).
